orbital4c/complex_fcn.py

```diff
@@ -68,8 +68,15 @@ def dot(self, other):
         im = vp.dot(self.real, other.imag) + vp.dot(self.imag, other.real)
         return re + 1j * im
       
-    def gradient(self):
-        D = vp.ABGVDerivative(self.mra, 0.0, 0.0)
+    def gradient(self, der = 'ABGV'):
```

Comment:
I don't understand why these changes appear in your PR. They belong to one of my previous PRs which you have approved. If you have rebased your code correctly this should not happen, I believe. I can try to fork your branch and see if I understand it.